TypeCobol/Compiler/Types/TypedefType.cs

```diff
@@ -65,7 +65,7 @@ internal override void SetFlag(Symbol.Flags flag, bool value)
         /// T Typedef S-> S Typedef R -> R PIC X(10).
         /// give the type of PIX X(10)
         /// </summary>
-        public override Type TypeComponent => TargetType?.TypeComponent ?? TargetType;
+        public override Type TypeComponent => TargetType;
```

Comment:
Does this kind of code will works `01 MyType typedef strict type Date.` ?

Reply:
Yes, the visitor will follow through the TypeComponent of the TargetType. In short, this logic is already contained within the visitor, no need to have it embedded in the Type.